code/modules/modular_computers/file_system/programs/security/penalmechs.dm

```diff
@@ -0,0 +1,128 @@
+/datum/computer_file/program/penal_mechs
+	filename = "penalmechs"
+	filedesc = "Remote Penal Monitoring"
+	program_icon_state = "security"
+	extended_desc = "This program allows monitoring and control of active penal miner mechs."
+	required_access_run = access_armory
+	required_access_download = access_armory
+	requires_ntnet = 1
+	available_on_ntnet = 1
+	network_destination = "penal mining mech monitoring system"
+	size = 11
+	usage_flags = PROGRAM_ALL_REGULAR
+	color = LIGHT_COLOR_ORANGE
+	nanomodule_path = /datum/nano_module/camera_monitor
```

Comment:
NO. JUST NO.

Nano module defines what ***Nano module*** handles ui rendering and topic. This is VueUI UI so this has to go.

Reply:
Scooted the necessary procs elsewhere.